cli/tests/pcluster/aws/test_elb.py

```diff
@@ -0,0 +1,194 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License'). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the 'LICENSE.txt' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from assertpy import assert_that
+
+from pcluster.aws.elb import ElbClient
+from tests.utils import MockedBoto3Request
+
+
+@pytest.fixture()
+def boto3_stubber_path():
+    return "pcluster.aws.common.boto3"
+
+
+@pytest.mark.parametrize("generate_error", [True, False])
+def test_list_load_balancers(boto3_stubber, generate_error):
+    """Verify that list_instance_types behaves as expected."""
+    dummy_load_balancer_arn = "dummy_load_balancer_arn"
+    dummy_load_balancer_arn_2 = "another_dummy_load_balancer_arn"
```

Comment:
These seem unused elsewhere, for consistencies with other test I'd remove these variables and use the strings below.